src/logic/collectibles/store/actions/fetchCollectibles.ts

```diff
@@ -4,16 +4,26 @@ import { getNetwork } from 'src/config'
 import { getConfiguredSource } from 'src/logic/collectibles/sources'
 import { addNftAssets, addNftTokens } from 'src/logic/collectibles/store/actions/addCollectibles'
 import { safeParamAddressFromStateSelector } from 'src/routes/safe/store/selectors'
+import { State } from '../../../../store'
+import { Dispatch } from 'redux'
 
-const fetchCollectibles = () => async (dispatch, getState) => {
-  const network = getNetwork()
-  const safeAddress = safeParamAddressFromStateSelector(getState()) || ''
-  const source = getConfiguredSource()
-  const collectibles = await source.fetchAllUserCollectiblesByCategoryAsync(safeAddress, network)
+const fetchCollectibles = () => async (dispatch: Dispatch, getState: () => State): Promise<void> => {
+  return new Promise(async (resolve) => {
```

Comment:
https://javascript.info/async-await#async-functions

> The word “async” before a function means one simple thing: a function always returns a promise. 

That means that wrapping it in a Promise is not necessary, it’s already returning a promise.

Reply:
fixed